### DISCUSSION TITLE
[Feature] Sort by Oldest Created Photos in a Created Album

### DISCUSSION THREAD

**Discussion by chk86** [immich-app/immich#4735] [Feature Request]
### The feature

**Problem:**
Immich defaults to "Newest First" when displaying photos within a created album. To view in chronological order by oldest created photo first, a user has to scroll all the way to the bottom and then scroll up.

**Requested Feature:**
Add a simple toggle or dropdown in albums, allowing users to sort photos within a created album by "Oldest First."

**Benefits:**
It gives users flexibility when viewing albums, especially when viewing in chronological order is important (e.g., family events, vacations, etc.)

**Implementation:**
1. Add a "Sort: Newest/Oldest" toggle within the album view.
2. Rearrange photos based on the user's choice.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #1689

**Reply by chk86** [reply]
The primary feature request in that request is the albums view page. The secondary request, sorting within an album, got buried. Reading through the comments, it's not clear this second request is still active, no?

**Comment by scottrbaxter** [comment]
This was closed as a duplicate, but seems to not be implemented as of `v1.92.1`. Should this be reopened?
